pycbc/coordinates/base.py

```diff
@@ -140,8 +141,3 @@ def spherical_to_cartesian(rho, phi, theta):
     y = rho * numpy.sin(phi) * numpy.sin(theta)
     z = rho * numpy.cos(theta)
     return x, y, z
-
```

Comment:
Don't remove this. It should explicitly say what should be imported by * which is what __all__ does. Your module should also use this feature.

Reply:
@ahnitz Will this be duplicated? Because `__init__.py` has `__all__` already.